docs/provider/devolutions-server.md

```
@@ -43,12 +44,41 @@ kubectl create secret generic dvls-credentials \
+| Entry name | `db-credentials` |
+| Entry name with folder path | `infrastructure/databases/db-credentials` |
+| Folder path with backslashes | `infrastructure\databases\db-credentials` |
 
-You can find these UUIDs in the DVLS web interface by viewing the entry properties.
+The vault is configured in the SecretStore's `vault` field (name or UUID), so the key only needs to identify the entry.
+
+### Folder paths
+
+If an entry is inside a folder, you can include the folder path before the entry name. Both forward slashes (`/`) and backslashes (`\`) are accepted as path separators:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)